### PR TITLE
[enhancement](cloud) add CloudTabletStatMgr to capture stats in cloud mode

### PR DESCRIPTION
**Pull request by freemandealer**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by freemandealer** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Comment by freemandealer** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/CloudTabletStatMgr.java [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+import java.util.concurrent.ForkJoinPool;
+
+/*
+ * CloudTabletStatMgr is for collecting tablet(replica) statistics from backends.
+ * Each FE will collect by itself.
+ */
+public class CloudTabletStatMgr extends MasterDaemon {
+    private static final Logger LOG = LogManager.getLogger(CloudTabletStatMgr.class);
+
+    private ForkJoinPool taskPool = new ForkJoinPool(Runtime.getRuntime().availableProcessors());
```
Comment (SWJTU-ZhangLei) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/CloudTabletStatMgr.java:49]: The `taskPool` seems redundant

**Review by swjtu-zhanglei — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**